sdk/js/src/Client.ts

```diff
@@ -118,11 +118,18 @@ export class DVCClient implements Client {
         }
 
         try {
+            const variableFromConfig = this.config?.variables?.[key]
+
             this.eventQueue.queueAggregateEvent({
                 type: variable.isDefaulted
                     ? EventTypes.variableDefaulted
                     : EventTypes.variableEvaluated,
                 target: variable.key,
+                metaData: {
+                    value: variable.value,
+                    type: variableFromConfig?.type,
```

Comment:
wouldn't this only work if the variable exists in the config? We wouldn't be sending the type if the variable used the default value